lib/Lift/LoadBinaryPass.cpp

```diff
@@ -40,3 +40,16 @@ bool LoadBinaryWrapperPass::runOnModule(llvm::Module &M) {
   BinaryView.emplace(*Model, Data);
   return false;
 }
+
+std::optional<std::pair<RawBinaryView, std::unique_ptr<llvm::MemoryBuffer>>>
+loadBinary(const model::Binary &Model, llvm::StringRef BinaryPath) {
+  revng_assert(Model.verify());
+
+  auto FileContentsBuffer = llvm::MemoryBuffer::getFileOrSTDIN(BinaryPath);
+  if (FileContentsBuffer.getError())
+    return std::nullopt;
```

Comment:
No reason to drop the error, return an `llvm::Expected<...>`

Reply:
Done: https://github.com/revng/revng/pull/173/commits/5b6f6182f7ecdbabf16eea4ebe6d3bd5121507d2 and https://github.com/revng/revng/pull/173/commits/2e0f17ec67ad1a9a5fb1ec3a0a5db2e740f31e73.